virttest/ssh_key.py

```diff
@@ -10,17 +10,29 @@
 from virttest import remote as remote_old
 
 
-def get_public_key():
+def get_public_key(client_user=None):
     """
     Return a valid string ssh public key for the user executing autoserv or
     autotest. If there's no DSA or RSA public key, create a RSA keypair with
     ssh-keygen and return it.
 
+    :param client_user: genenrate the ssh_key for which client
```

Comment:
usually we prefer one line doc string for one parameter. Do you intend to use multiple lines ?